NPE in N4JSC when srcFiles is not provided

```
09:33:00 [build] Caused by: java.lang.NullPointerException
09:33:00 [build] 	at org.eclipse.n4js.hlc.base.DependenciesHelper.convertToFilesAddTargetPlatformAndCheckWritableDir(DependenciesHelper.java:158)
09:33:00 [build] 	at org.eclipse.n4js.hlc.base.DependenciesHelper.discoverMissingDependencies(DependenciesHelper.java:54)
09:33:00 [build] 	at org.eclipse.n4js.hlc.base.N4jscBase.doMain(N4jscBase.java:507)
09:33:00 [build] 	at org.eclipse.n4js.hlc.base.N4jscBase.main(N4jscBase.java:325)
09:33:00 [build] 	... 11 more
```

--- a/tools/org.eclipse.n4js.hlc.base/src/org/eclipse/n4js/hlc/base/DependenciesHelper.java
+++ b/tools/org.eclipse.n4js.hlc.base/src/org/eclipse/n4js/hlc/base/DependenciesHelper.java
@@ -29,6 +29,7 @@ import org.eclipse.n4js.projectModel.IN4JSProject;
 import org.eclipse.n4js.projectModel.dependencies.DependenciesCollectingUtil;
 
 import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
 import com.google.common.base.Throwables;
 import com.google.common.collect.Sets;
 import com.google.inject.Inject;
@@ -157,7 +158,7 @@ class DependenciesHelper {
 			FileUtils.isExistingWriteableDir(tpLoc);
 			retList.add(tpLoc);
 		}
-		if (!dirpaths.isEmpty()) {
+		if (!Strings.isNullOrEmpty(dirpaths)) {
 			for (String dirpath : Splitter.on(File.pathSeparatorChar).split(dirpaths)) {
 				final File ret = new File(dirpath);
 				FileUtils.isExistingWriteableDir(ret);

